tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -209,11 +213,11 @@ local utils = import "templates/utils.libsonnet";
     accelerator: tpus.v3_8,
   },
   configs: [
-    hf_glue + v3_8 + bert_base_cased + timeouts.Hours(1),
-    hf_glue + v2_8 + bert_base_cased + timeouts.Hours(1),
-    hf_glue + v3_8 + xlnet_large_cased + timeouts.Hours(3),
-    hf_glue + v3_8 + roberta_large + timeouts.Hours(3),
-    hf_glue + v3_8 + xlm_mlm_en_2048 + timeouts.Hours(4),
-    hf_glue + v3_8 + distilbert_base_uncased + timeouts.Hours(1),
+    hf_glue + v3_8 + bert_base_cased + timeouts.Hours(2),
```

Comment:
why the increase? are you seeing slowdowns?

Reply:
Yeah mainly at the dataset setup stage, which was refactored by HF. But training time wise no, no slowdown afaict.